x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_count_panel/alerts_count.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiProgress, EuiBasicTableColumn, EuiInMemoryTable } from '@elastic/eui';
+import React, { memo, useMemo } from 'react';
+import styled from 'styled-components';
+import numeral from '@elastic/numeral';
+import { useUiSetting$ } from '../../../../common/lib/kibana';
+import { DEFAULT_NUMBER_FORMAT } from '../../../../../common/constants';
+import * as i18n from './translations';
+import { DefaultDraggable } from '../../../../common/components/draggables';
+import { GenericBuckets } from '../../../../../common';
```

Comment:
NIT: I was told to use `import type { ...` when possible, to reduce the plugin size